### ISSUE TITLE
Loss Function in gen.ipynb

### ISSUE THREAD

**Issue by mitkina** [muratsensoy/muratsensoy.github.io#1]
Hello,

I had a question about the following loss function in gen.ipynb (Eq. (4) in the paper):

```
def loss_fn(Y, evidence, real_p, fake_p):
    disc_loss = tf.reduce_mean(tf.reduce_sum(-Y * tf.log(real_p + 1e-5), axis=1) +
                               tf.reduce_sum(-Y * tf.log((1.0 - fake_p) + 1e-5), axis=1))
    alp = misleading_alpha(evidence+1, Y)
    disc_loss += tf.reduce_mean(KL(alp))
    return disc_loss
```

Should the second term in disc_loss be: 
```
disc_loss = tf.reduce_mean(tf.reduce_sum(-Y * tf.log(real_p + 1e-5), axis=1) +
                               tf.reduce_sum(-(1-Y) * tf.log((1.0 - fake_p) + 1e-5), axis=1))
```

And just for my better understanding, does Y == 0 when the data is just from out of distribution or also when the class is incorrect?

Thank you!

**Comment by hnyz979** [comment]
Hi.
I also have a question about the "misleading_alpha" function. It seems that this corresponds to Equation 5 in the paper. However, in Equation 5 it writes "p_{−k} refer to the vector of probabilities p_j such that j ≠ k", why should we use "tf.where(tf.equal(y, tf.constant(0, dtype=tf.float32)))" in the "misleading_alpha" function? 
Thanks~
